content/more/code_of_conduct.md

```diff
@@ -0,0 +1,54 @@
+# Code of Conduct
+
+We follow the [Carpentries' Code of
+Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html)
+to provide an inclusive and supportive environment for our learners and
+our members. Any violation of our Code of Conduct should be reported, it
+will then be handled as quickly as possible and will remain
+confidential:
+
+- to report Code of Conduct violations for official Carpentries
+    workshops, see the Carpentries [Code of Conduct Incident Reporting
+    Guidelines](https://docs.carpentries.org/topic_folders/policies/incident-reporting.html)
+- to report Code of Conduct violations for local Carpentry@UiO
+    workshops, use UiO's
+    [speak-up](https://www.uio.no/english/about/hse/speak-up/index.html)
+    (English) or [si-fra](https://www.uio.no/om/hms/si-fra/) (Norwegian)
+    reporting
+
+If the workshop is a local workshop rather than an official Carpentries
+workshop, the workshop website will have the Carpentry@UiO logo:
+
+![Carpentry@UiO logo](uio-carpentry-logo-small.png)
```

Comment:
This image does not show on https://teebusch.github.io/uio-carpentry.github.io/more/code_of_conduct/

Reply:
Yes, there's an issue with relative image paths in Hugo that I haven't figured out yet.